TESTING.md

```diff
@@ -140,4 +140,24 @@ mypy .
 
 from the root. See `mypy.ini` for our custom mypy settings.
 
+## Generate documentation from Jupyter notebooks
 
+The documents in the `tutorials` section are automatically generated from the Jupyter notebooks.
+
+The `convert.sh` script in the `scripts/nb2doc/` directory can be used:
+
+* to generate documents from new notebooks;
+* to ensure that any changes to the existing notebooks are reflected in the existing documents (for instance in a CI setting).
```

Comment:
Would it be possible to run the script before building the documentation?
As in, we dont need to checkin the generated `adoc` files but only generate them on the fly

Reply:
Good point. Failing on a diff seems indeed be better then